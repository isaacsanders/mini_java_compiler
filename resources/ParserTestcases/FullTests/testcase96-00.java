class Evil extends Satan {
    public static void main (String[] ID){
	return ID;
    }

    public boolean zombies(){
	return true;
    }
}

class LilEvil extends Satan {
    public static void main (String[] ID){
    }
    public void heavenlyGrace(){
	// returns absolutely nothing, of course
    }
}