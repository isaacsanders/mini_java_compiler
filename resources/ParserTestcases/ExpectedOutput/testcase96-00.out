

ERROR in line 1 and column 12: Syntax error



ERROR in line 1 and column 12: Couldn't repair and continue parse

Exception processing file: Can't recover from previous error(s)
